integration-test/src/test/scala/com/cloudera/livy/test/BatchIT.scala

```diff
@@ -64,7 +80,7 @@ class BatchIT extends BaseIntegrationTestSuite {
 
   pytest("submit a pyspark application") {
     val hdfsPath = uploadResource("pytest.py")
-    val output = newOutputPath()
+    val output = newOutputPath
```

Comment:
What you are suggesting is against the recommended Scala style but I don't have a strong opinion on this. Will fix.